app_clients/pod-delete/pod-delete.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pod-delete
```

Comment:
use run-id here, refer other experiments

Reply:
We don't do that here in test tools it is already added in pod spec in experiment chaoslib 
ref: https://github.com/litmuschaos/test-tools/blob/master/resource_stress/app-cpu-stress/app-cpu-stress.yml